staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1308,6 +1309,56 @@ func (c *Cacher) Ready() bool {
 	return ok
 }
 
+func (c *Cacher) ShouldDelegateExactRV(resourceVersion string, recursive bool) (delegator.Result, error) {
+	// Recursive not supported unitl exact RV is implemented for WaitUntilFreshAndGet.
+	if !recursive || c.watchCache.snapshots == nil {
+		// Recursive not supported until WaitAnd
```

Comment:
nit: remove (it's a partial duplicate of the comment above)

Reply:
Done